webgrid/__init__.py

```diff
@@ -807,7 +763,7 @@ class ColumnGroup(object):
 
     Args:
         label (str): Grouping label to be rendered for the column set.
-        class\_ (str): CSS class name to apply in HTML rendering.
+        class_ (str): CSS class name to apply in HTML rendering.
```

Comment:
I'm pretty sure the \ escape was needed for something. Check the sphinx output against https://webgrid.readthedocs.io/en/latest/columns/base-column.html#webgrid.ColumnGroup

That made me look to see if we have a docs check in webgrid, like https://github.com/level12/keg-auth/blob/master/tox.ini#L58, and we don't... It needs to be there; please add that block for tox and make sure it passes.

Reply:
Unsurprisingly, adding the docs check made it fail, so I put the escapes back.